tests/torch/fx/test_model_transformer.py

```diff
@@ -136,6 +152,78 @@ def test_output_insertion_transformation(tuple_output, target_point):
     )
 
 
+def torchvision_model_case(model_id: str, input_shape: Tuple[int,]):
+    model = getattr(models, model_id)
+    return ModelCase(partial(model, weights=None), model_id, input_shape)
+
+
+TEST_MODELS_QUANIZED = (
+    (torchvision_model_case("resnet18", (1, 3, 224, 224)), {}),
+    (torchvision_model_case("mobilenet_v3_small", (1, 3, 224, 224)), {}),
+    (torchvision_model_case("vit_b_16", (1, 3, 224, 224)), {"model_type": nncf.ModelType.TRANSFORMER}),
+    (torchvision_model_case("swin_v2_s", (1, 3, 224, 224)), {"model_type": nncf.ModelType.TRANSFORMER}),
+)
```

Comment:
Maybe it worth to test only one synthetic model here? This will increase time significantly ( there will be x3 model quantizations than before). Besides, new supported models should be specified in `test_models` and here, and this approach looks ambiguous
CC: @alexsu52

Reply:
Alright, I could test only a resnet model to save time.

Reply:
Does it covers all the cases?

Reply:
it covers most of the cases except `quantize_per_tensor` node following a constant node.

Reply:
> Maybe it worth to integrate this tests into `test_models`? So, after the model is being checked by NNCFGraph we are checking its compressed weights?

Since this is a transformation test and checks if the backend parameters are working correctly too, I had decided to put it in `model_transformer_test`. Should I move it to `test_models`?

Reply:
It looks a little redundant, I will move it to `test_models` inside of `test_quantized_model`